src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,109 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (name, licenseNumber) VALUE(?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement driverPreparedStatement =
+                        connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            driverPreparedStatement.setString(1, driver.getName());
+            driverPreparedStatement.setString(2, driver.getLicenseNumber());
+            driverPreparedStatement.executeUpdate();
+            ResultSet resultSet = driverPreparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create driver in DB: "
+                    + driver, e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String query = "SELECT * FROM drivers WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getDriverFromDB = connection.prepareStatement(query)) {
+            getDriverFromDB.setLong(1, id);
+            ResultSet resultSet = getDriverFromDB.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(parseDriver(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't find driver into DB where id = "
+                    + id, e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't get driver with id = "
                    + id, e);
```

We get this exception while smth went wrong while we getting driver from db, but it could exist in db